### PR TITLE
poac 0.8.0

### PR DESCRIPTION
**Pull request by ken-matsui**
Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>## What's Changed
* Add install make command & version from poac.toml by @ken-matsui in https://github.com/poac-dev/poac/pull/802
* Automatically propagate the version in poac.toml by @ken-matsui in https://github.com/poac-dev/poac/pull/803
* reorganize `#include`s by @yaito3014 in https://github.com/poac-dev/poac/pull/804
* Use contains instead of count for code clarity by @ken-matsui in https://github.com/poac-dev/poac/pull/805
* Rename CFLAGS to CXXFLAGS by @ken-matsui in https://github.com/poac-dev/poac/pull/806
* Improve edition validation by @ken-matsui in https://github.com/poac-dev/poac/pull/807
* Use emplace_back instead of push_back by @ken-matsui in https://github.com/poac-dev/poac/pull/808
* Remove redundant path for includes and lint improvements by @ken-matsui in https://github.com/poac-dev/poac/pull/809
* Avoid using `using namespace` for string literals by @ken-matsui in https://github.com/poac-dev/poac/pull/810
* Simplify BuildConfig by @ken-matsui in https://github.com/poac-dev/poac/pull/811
* Implement semver parser & improve binary dependency analysis by @ken-matsui in https://github.com/poac-dev/poac/pull/812
* use usize by @yaito3014 in https://github.com/poac-dev/poac/pull/814
* Support parallel builds by @ken-matsui in https://github.com/poac-dev/poac/pull/816
* Reduce test outputs by @ken-matsui in https://github.com/poac-dev/poac/pull/817
* Support finding poac.toml in parent directories by @ken-matsui in https://github.com/poac-dev/poac/pull/818
* Validate git dependencies by @ken-matsui in https://github.com/poac-dev/poac/pull/819
* Enable parallel tests by @ken-matsui in https://github.com/poac-dev/poac/pull/820
* Implement VersionReq parser by @ken-matsui in https://github.com/poac-dev/poac/pull/821
* Bump actions/checkout from 3 to 4 by @dependabot in https://github.com/poac-dev/poac/pull/823
* Improve keyword handling by @ken-matsui in https://github.com/poac-dev/poac/pull/825
* Correctly handle build dependencies by @ken-matsui in https://github.com/poac-dev/poac/pull/826
* Support system dependencies by @ken-matsui in https://github.com/poac-dev/poac/pull/827

## New Contributors
* @yaito3014 made their first contribution in https://github.com/poac-dev/poac/pull/804

**Full Changelog**: https://github.com/poac-dev/poac/compare/0.7.0...0.8.0</pre>
</details>

### CODE REVIEW

File: Formula/p/poac.rb [hunk 1]
```diff
@@ -16,6 +16,9 @@ class Poac < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "902689bc9c40c5d9ac2c51fa4006f45ad7703a8c3f1cec033d2f0fd4d9c6012b"
   end
 
+  depends_on "libgit2"
+  depends_on "pkg-config"
```
Comment (p-linnane) [inline Formula/p/poac.rb:20]:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "libgit2"
```

**Review by p-linnane — CHANGES_REQUESTED** [review]

File: Formula/p/poac.rb (hunk 1 above)
Comment (ken-matsui) [reply Formula/p/poac.rb:20]: @p-linnane pkg-config is also a runtime dependency, but do we need to make this change?

https://github.com/poac-dev/poac/tree/0.8.0?tab=readme-ov-file#runtime-requirements

Comment (p-linnane) [reply Formula/p/poac.rb:20]: My apologies. I don't believe we have any existing packages that need `peg-config` at runtime, so I assumed that was an error. Thanks for letting me know.

**Review by p-linnane — APPROVED** [review]

File: Formula/p/poac.rb (hunk 1 above)
Comment (ken-matsui) [reply Formula/p/poac.rb:20]: No worries! Thank you for your review!

Comment (p-linnane) [reply Formula/p/poac.rb:20]: I stand corrected! Turns out `poac` will be the 17th formulae with a `pkg-config` runtime dependency.

**Pull request merged**